install.sh

```diff
@@ -22,6 +23,17 @@ download_dot_env_file() {
   "${Curl[@]}" "$RepoUrl"/example.env -o ./.env
 }
 
+generate_random_password() {
+  echo "Generate random password for .env file..."
+  # rand_pass=$(openssl rand -base64 20 | tr -dc A-Za-z0-9 | head -c10)
```

Comment:
I don't think we need to keep the other implementation commented here, lets just remove it if we're going with the other, more compatible implementation
```suggestion
```